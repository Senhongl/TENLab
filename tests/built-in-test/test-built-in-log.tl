a = [[10,11,12],[14,15,16]];
print(log(a));
b = [1.0,3.2,5.6];
print(log(b));