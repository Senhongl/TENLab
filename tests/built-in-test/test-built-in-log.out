 2.3026  2.3979  2.4849
 2.6391  2.7081  2.7726
[ CPUDoubleType{2,3} ]
 0.0000
 1.1632
 1.7228
[ CPUDoubleType{3} ]
